src/openforms/submissions/api/viewsets.py

```diff
@@ -198,6 +200,15 @@ def _complete(self, request, *args, **kwargs):
 
         logevent.form_submit_success(submission)
 
+        rules = FormLogic.objects.filter(form=submission.form)
```

Comment:
I decided to log the not triggered rule just before to submit the submission. If I did in the evaluation logic, I would have had potentially  the same logic rule triggered and not triggered in different logs